index.js

```diff
@@ -92,7 +121,7 @@ Dota2Client.prototype.launch = function() {
   this.Party = null;
   this.Lobby = null;
   this.PartyInvite = null;
-  this._client.gamesPlayed([this._appid]);
+  this._user.gamesPlayed([{"game_id": this._appid}]);
```

Comment:
should update this on the exit() method too (https://github.com/RJacksonm1/node-dota2/blob/3da89590521a2a8d8bb8647112aaff0a1109dc2f/index.js#L148)